rpmatch() is missing in uclibc so we provide a local definition
in form of a macro

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: git/src/utils/prompt.h
===================================================================
--- git.orig/src/utils/prompt.h	2012-06-06 15:51:00.899112775 -0700
+++ git/src/utils/prompt.h	2012-06-06 15:52:17.003116762 -0700
@@ -172,4 +172,12 @@
   return errstr;
 }
 
+#ifdef __UCLIBC__
+#define rpmatch(line) \
+       ( (line == NULL)? -1 : \
+         (*line == 'y' || *line == 'Y')? 1 : \
+         (*line == 'n' || *line == 'N')? 0 : \
+         -1 )
+#endif /* __UCLIBC__ */
+
 #endif /*ZYPPERPROMPT_H_*/
